OKTA-393339: add a reminder about postLogoutRedirectUri property (#314)
okta/okta-spring-boot: README.md

| okta.oauth2.audience   | api://default | The audience of your [Authorization Server](/docs/how-to/set-up-auth-server.html) |
| okta.oauth2.groupsClaim | groups | The claim key in the Access Token's JWT that corresponds to an array of the users groups. |
| okta.oauth2.postLogoutRedirectUri | N/A | Set to an absolute URI to enable [RP-Initiated (SSO) logout](https://developer.okta.com/blog/2020/03/27/spring-oidc-logout-options). |

> **NOTE**: by specifying the **postLogoutRedirectUri**, you will be redirected to it after the end of your session, therefore this resource must be publicly available so don't forget to add it to your HttpSecurity configuration:

Reviewer: ```suggestion
**NOTE**: On specifying the **postLogoutRedirectUri**, you will be redirected to it after the end of your session, therefore this resource must be publicly available so do not forget to add it to your `HttpSecurity` configuration:
```

```yaml
okta:
  oauth2:
    postLogoutRedirectUri: "http://localhost:8080/logout/callback"
```
```java
import org.springframework.context.annotation.Configuration;
import org.springframework.security.config.annotation.web.builders.HttpSecurity;
import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;

@Configuration
public class WebConfig extends WebSecurityConfigurerAdapter {
    @Override
    protected void configure(HttpSecurity http) throws Exception {
        http.authorizeRequests()

>             // allow antonymous access to the root and logout pages

Reviewer: ```suggestion
            // allow anonymous access to the root and logout pages
```
